src/app/ApplicationTemplate.Shared.Views/Startup.cs

```diff
@@ -113,9 +113,9 @@ protected override async Task StartViewServices(IServiceProvider services, bool
 
 	private static async Task SetShellViewModel()
 	{
-		await App.Instance.Shell.Dispatcher.RunAsync((CoreDispatcherPriority)CoreDispatcherPriority.Normal, SetDataContextUI);
+		App.Instance.Shell.DispatcherQueue.TryEnqueue(DispatcherQueuePriority.Normal, SetDataContextUI);
```

Comment:
This is not equivalent to the previous code because the dispatcher instruction is not awaited.

Reply:
If I understand correctly by doing a quick research, this API related, but `Dispatcher` and `DispatcherCore` are linked to `CoreWindow` which WinUI doesn't have so they are both `null`.

Reply:
What I meant is that you probably want an awaitable equivalent of `RunAsync` but for `DispatcherQueue`. If you need a reference, extensions methods were made in a few packages like Chinook.Navigation and Nventive.View.